lms/djangoapps/discussion/rest_api/discussions_notifications.py

```diff
@@ -447,7 +448,9 @@ def clean_thread_html_body(html_body):
     """
     Get post body with tags removed and limited to 500 characters
     """
-    html_body = BeautifulSoup(Truncator(html_body).chars(500, html=True), 'html.parser')
+    truncated_body = Truncator(html_body).chars(500, html=True)
```

Comment:
`Truncator` escapes quotes to encoding,  `match.unwrap()` escapes it again and in this way string was escaping two times, that's why we are unescaping here.